code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts

```diff
@@ -23,26 +23,38 @@ import type { BuilderOptions, TypescriptOptions } from '../types';
 import { createBabelLoader } from './babel-loader-preview';
 
 const storybookPaths: Record<string, string> = {
-  global: path.dirname(require.resolve(`global/package.json`)),
+  global: dirname(require.resolve(`global/package.json`)),
   ...[
-    'addons',
+    //
```

Comment:
```suggestion
```

Reply:
I did this because these comments ensure the list stays on multiple lines, irregardless of how many items are in it, making it easier to move them between the 2.